content/docs/tutorials/interactivity/wrapping-up.md

```diff
@@ -0,0 +1,18 @@
+---
+$title: Wrapping up
+$order: 4
+---
+
+Congratulations! You just created an interactive e-commerce product detail page with variable sizes and prices for each SKU, fetched on-demand from a remote JSON endpoint.
+
+You learned how to:
+
+- Use `<amp-state>` to fetch product information (color, size, availability).
+- Use `<amp-bind>` to create cross-element interactions.
+- Create a fast and rich user experience with AMP!
+
+If you got stuck, see the `static/final.html` file for the complete solution.
```

Comment:
link to https://github.com/googlecodelabs/advanced-interactivity-in-amp/blob/master/static/final.html